SystemServices/SystemServices.cpp

```diff
@@ -467,7 +467,7 @@ namespace WPEFramework {
             registerMethod(_T("getWakeupReason"), &SystemServices::getWakeupReason, this);
             registerMethod(_T("getLastWakeupKeyCode"), &SystemServices::getLastWakeupKeyCode, this);
 #endif
-            registerMethod("uploadLogs", &SystemServices::uploadLogs, this);
+            registerMethod("uploadLogs", &SystemServices::uploadLogsAsync, this);
```

Comment:
Since the uploadLogs() function is not used in the code, that can be removed. Also check any other functions used inside uploadLogs() that are not used any where can be removed.